docs/source/getting_started/installation.md

```diff
@@ -5,12 +5,52 @@ installation of the plugin and its dependencies. `uv` provides advanced
 dependency resolution which is required to properly install dependencies like
 `vllm` without overwriting critical dependencies like `torch`.
 
-```bash
-# Install uv
-pip install uv
-
-# Install vllm-spyre
-git clone https://github.com/vllm-project/vllm-spyre.git
-cd vllm-spyre
-VLLM_TARGET_DEVICE=empty uv pip install -e .
-```
+1. Clone vllm-spyre
+
+   ```sh
+   git clone https://github.com/vllm-project/vllm-spyre.git
+   cd vllm-spyre
+   ```
+
+1. Install uv
+  
+   ```sh
+   pip install uv
+   ```
+  
+1. Create a new env
+
+   ```sh
+   uv venv --python 3.12 --seed .venv
+   ```
+
+1. Activate it
+  
+   ```sh
+   source .venv/bin/activate
+   ```
+
+1. Install `vllm-spyre` locally with dev (and optionally lint) dependencies
+  
+   ```sh
+   uv sync --frozen --active
+   ```
+  
+   or also with lint:
+  
+   ```sh
+   uv sync --frozen --group lint
```

Comment:
I think `--inexact` can also be used here so that it doesn't cull out other things like `xgrammar` and `torch`

Reply:
Yeah I thought about it, since it's a fresh environment I was assuming that there is nothing installed yet. But I guess it's safe to add it though

Reply:
Although I think this makes sense for the lint PR too? 

https://github.com/vllm-project/vllm-spyre/pull/175/files What do you think?